### DISCUSSION TITLE
Windows 11 Defender Antivirus complaining

### DISCUSSION THREAD

**Discussion by MWinhard** [1Remote/1Remote#391] [Show and tell]
Just want to let you know:

Last week Defender Antivirus on my Windows 11 VM suddenly decided that it's time to detect a threat in 1Remote.exe. This version of 1Remote is in use on this VM for several weeks. So I'm quite sure it's a false positive.

![image](https://user-images.githubusercontent.com/111646687/231134750-818b3a82-2141-484f-a853-34a0df6e10f8.png)

Unfortunately I have no idea why it complains. The linked Defender help site isn't of any help.
[https://www.microsoft.com/en-us/wdsi/threats/malware-encyclopedia-description?name=Trojan%3aScript%2fWacatac.B!ml&threatid=2147735503](https://www.microsoft.com/en-us/wdsi/threats/malware-encyclopedia-description?name=Trojan%3aScript%2fWacatac.B!ml&threatid=2147735503)

This is the version I'm using since several weeks:

![image](https://user-images.githubusercontent.com/111646687/231134993-aeefd603-4417-45b1-8d41-954f0eafccca.png)

**Comment by majkinetor** [comment]
Meh... AV is a disaster, you need team of people not to be a virus nowdays.

Perhaps due to startup?

**Comment by MWinhard** [comment]
It may be an additional factor, that 1Remote.exe is running on startup. 
But anyway. I'm quite sure, MS Defender would have caught it on manual start, too.

**Comment by majkinetor** [comment]
So, the offical way is to go to AV site, report it as false positive, and do that for each AV that complains.  Anybody can do that.

I, personally, have better things to do...

**Comment by majkinetor** [comment]
https://answers.microsoft.com/en-us/windows/forum/all/how-can-i-know-if-wacatacbml-is-a-false-positive/33107e01-d390-4579-aa94-625a3402d26a

**Comment by VShawn** [comment]
I know few about virus detection 

but since we can not submit a sample with personal account, let's just let this false positive go.
